Automattic/jetpack
# Shared UI for wordads

This patch creates an independent, shared UI for WordAds, inspired by Jetpack Search. 

![wordads-basic-ui](https://user-images.githubusercontent.com/51896/159819042-b3c97454-bd18-4be0-8371-97907d11025f.jpg)

#### Changes proposed in this Pull Request:

* New shared UI for WordAds, disabled by default in the production build of Jetpack (for now)
* Update Search UI to use new shared `Gridicon` component

#### Jetpack product discussion

The goals of centralising and rewriting the UI are as follows:
- Single UI codebase for Jetpack and Calypso (current Calypso implementation is very out-of-sync with Jetpack implementation)
- Foundation for standalone WordAds plugin, if we go that route
- Enables users to view all their WordAds information (placements, payments, etc) in one place, rather than having to go to Calypso to do some of those things
- Much more modern experience using up-to-date styleguides, fonts, etc.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Edit your wp-config.php and add the line `define( 'ENABLE_WORDADS_SHARED_UI', true );`. By default this will be in `tools/docker/wordpress/wp-config.php`
* Ensure Jetpack is connected (the UI doesn't currently detect connection status or prompt to connect Jetpack)
* Go to https://goldsounds.jurassic.tube/wp-admin/admin.php?page=jetpack-wordads
* Check it looks like the screenshot at the top of this description.
* Don't worry if the toggle doesn't actually work.
